shumgrepper/__init__.py

```diff
@@ -160,14 +160,14 @@ def tar_sum(tar_sum):
 # request files by tarsum
 @app.route('/tarball/<tarball>/filenames')
 def tarball_filenames(tarball):
-    messages = sm.File.by_tarball(session, tarball)
+    filenames = sm.File.tarball_filenames(session, tarball)
 
-    file_list = map(lambda x: x.filename, messages)
+    filename_list = [filename[0][1:] for filename in filenames]
```

Comment:
@pypingou  Because we get a list of tuples from querying. To get first element of that tuple I did [0]. 
The filename starts with a '/' and to call /filename, I have to remove this slash, so did [1:]